twa

```diff
@@ -440,6 +440,16 @@ function stage_5_robots_check {
   code=$(fetch_respcode "${domain}/robots.txt")
   if [[ "${code}" -eq 200 ]]; then
     PASS "Site provides robots.txt at: ${domain}/robots.txt"
+    robots=$(fetch -L -s "${domain}/robots.txt")
+
+    # set nocasematch option
+    shopt -s nocasematch
+
+    if [[ $robots =~ .*(admin|cgi-bin).* ]]
+    then
+       MEH "Robots.txt might be exposing some sensitive directories"
```

Comment:
Nit: Don't capitalize `robots.txt` here.

Let's tweak the language here too: maybe "robots.txt lists CGI scripts and/or admin panels"? Might be worth breaking the two main cases (CGI scripts, admin panels) into separate checks.